_data/meltano/utilities/elementary/elementary.yml

```
@@ -0,0 +1,80 @@
+name: elementary
+label: Elementary
+domain_url: https://docs.elementary-data.com/introduction
+namespace: elementary
+pip_url: elementary-data[bigquery]=0.5.3 git+https://github.com/potloc/elementary-ext.git
+maintenance_status: active
+repo: https://github.com/elementary-data/elementary
+executable: edr
+logo_url: /assets/logos/utilities/elementary.png
+variant: elementary
+description: Elementary is an open-source data observability solution for data & analytics engineers.
+
+settings:
+- name: profiles-dir
```

```suggestion
- name: profiles-dir
  label: Profiles Directory
```

@SBurwash  we'll need labels on each of the settings.